resources/ory/charts/hydra/charts/hydra-maester/templates/deployment.yaml

```diff
@@ -31,13 +31,16 @@ spec:
           command:
             - /manager
           args:
-            - --metrics-addr=127.0.0.1:8080
+            - --metrics-addr=:{{ .Values.port.metrics }}
```

Comment:
Consider changing it to `0.0.0.0:port` for more clarity

Reply:
thanks, done!